src/main/java/seedu/address/model/Model.java

```diff
@@ -109,6 +109,13 @@ public interface Model {
      */
     void addModule(Module module);
 
+    /**
+     * Replaces the given module {@code target} with {@code editedModule}.
+     * {@code target} must exist in the address book.
+     * The module given in {@code editedPerson} must not be the same as another existing module in the address book.
```

Comment:
```suggestion
     * The module given in {@code editedModule} must not be the same as another existing module in the address book.
```
Minor lapse in comment

Reply:
Thanks! Have adjusted accordingly